File: vector_length_finder.java
//what up from Chernobyl
import java.util.Scanner;
import java.lang.Math;
class vector_length_finder{
    static double find_length(double vect[] , int array_size){
        double length = 0;
        for ( int lol = 0 ; lol < array_size ; lol++ ){
            length += Math.pow(vect[lol],2);
        }
        return Math.sqrt(length);
    }
    public static void main(String[] args){
        Scanner read = new Scanner(System.in);
        System.out.print("Enter desiered space:\t");
        int s = read.nextInt();

        double vec_cords[] = new double[s];
        System.out.println("Enter vector's coordinates:");
        for ( int lol = 0 ; lol < s ; lol++ ){
            vec_cords[lol] = read.nextDouble();
        }
        double length = find_length(vec_cords , s);

        if ( length == 1 ){
            System.out.println(" =>\n" + "||V|| = " + (length) );
        }
        else {
            System.out.println( " =>\n" + "||V|| = " + (length) + "\n");
            System.out.println( "Length != 1 , Rectifing....");
            System.out.print( "^\n" + "V = " + "(" );
            for ( int lol = 0 ; lol < s ; lol++ ){
                vec_cords[lol] /= length;
                System.out.print( (vec_cords[lol]) + ( lol < (s-1) ? " , " : "" ) );
            }
            System.out.println( ")\n" + "  =>\n" + "||V || = " + (find_length(vec_cords , s)) );

        }

    }
}

File: vector_length_finder.cpp
//what up from Chernobyl
#include <stdio.h>
#include <math.h>
double find_length(double vector[] , int array_size);

int main(){
    
    printf("Enter desiered space:\t");
    int space;
    scanf("%d", &space);

    double vector_coordinates[space];
    puts("Enter vector's coordinates:");
    
    for ( int lol = 0 ; lol < space ; lol++ ){
        scanf("%lf", &vector_coordinates[lol]);
    }
    
    double length = find_length(vector_coordinates , space);

    if ( length == 1 ){
        printf(" ->\n||V|| = %f", length);
    }
    else {
        printf(" ->\n||V|| = %f\n", length);
        puts("Length != 1 , Rectifing....");
        printf( "^\nV = (");
        
        for ( int lol = 0 ; lol < space ; lol++ ){
            vector_coordinates[lol] /= length;
            printf("%f%s", (vector_coordinates[lol]) , ( lol < (space-1) ? " , " : "" ) );
        }
        printf( ")\n  ->\n||V || = %f\n", find_length(vector_coordinates , space) );

    }
}

double find_length(double vector[] , int array_size){
    double length = 0;
    for ( int lol = 0 ; lol < array_size ; lol++ ){
        length += pow(vector[lol],2);
    }
    return sqrt(length);
}
